src/snowflake/cli/_plugins/streamlit/commands.py

```diff
@@ -81,6 +81,18 @@
 )
 
 
+@app.command(requires_connection=True)
+def execute(
+    name: FQN = StreamlitNameArgument,
+    **options,
+):
+    """
+    Executes a streamlit in a headless mode.
+    """
+    _ = StreamlitManager().execute(app_name=name)
+    return MessageResult(f"Streamlit {name} executed.")
```

Comment:
Usually we return message from Snowflake, in your case, what message is returned?

Reply:
After running "snow execute streamlit <my-app>" the message was "Streamlit <my-app> executed."

Reply:
I've added the integration test as requested. Apologies for the high number of commits—I had trouble running the tests locally and could only view the results after pushing to the branch.